php/public/style.css

```diff
@@ -1,34 +1,47 @@
 html, body {
     padding: 0;
     margin: 0;
+    height: 100%;
+    width: 100%;
+    overflow: clip;
+    background-color: #0082c9;
+    background-image: linear-gradient(40deg, #0082c9 0%, #30b6ff 100%);
     font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Cantarell, Ubuntu, Helvetica Neue, Arial, Noto Color Emoji, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;;
 }
 
+h1, h2, p {
+    margin: 8px 0 0;
+    text-align: justify;
+}
+h1, h2 {
+    margin-top: 16px;
+}
+
 a {
     text-decoration: none;
     color: #0082c9;
 }
 
 a.button,
 input[type="submit"] {
-    padding: 6px 16px;
+    padding: 8px 16px;
     width: auto;
-    min-height: 34px;
+    height: 34px;
     cursor: pointer;
     background-color: #0082c9;
     font-weight: bold;
-    border-radius: 100px;
-    margin: 3px 3px 3px 0;
-    font-size: 13px;
+    border-radius: 8px;
+    margin: 0;
+    font-size: 14px;
     color: white;
-    border: 1px solid black;
+    border: .5px solid black;
     outline: none;
 }
 
 a.button:focus,
 input[type="submit"]:focus {
-    color: black;
-    border: 2px solid black;
+.button:focus {
+    border: 1px solid black;
 }
```

Comment:
The `.button:focus` selector is redundant to `a.button:focus`.
I'm not sure the `.button:focus` selector is useful...